src/main/java/seedu/modtracker/ModTracker.java

```diff
@@ -4,18 +4,31 @@
  * Main entry-point for the java.seedu.modtracker application.
  */
 public class ModTracker {
-    private static ModuleList modules;
+
+    private Ui ui;
+    private final ModuleList modules = new ModuleList();
```

Comment:
I think this line should be in the main class instead of being declared outside

Reply:
I declare outside because it is used in more than one methods.